docs/detections/alerts-add-to-cases.asciidoc

```diff
@@ -28,7 +28,7 @@ meet the necessary <<case-permissions,prerequisites>>.
 . Click *Create case* after you've completed all of the required fields. A confirmation message is displayed with an option to view the new case. Click the link in the notification or go to the Cases page to view the case.
 
 [role="screenshot"]
-image::images/add-alert-to-new-case.png[Create case flyout with sample data filled in]
+image::images/add-alert-to-new-case.png[width=90%][height=90%][Create case flyout with sample data filled in]
```

Comment:
```suggestion
image::images/add-alert-to-new-case.png[width=60%][height=60%][Create case flyout with sample data filled in]
```

Maybe scale down the image even more? It couldn't even fit into my browser without scrolling:

<img width="1600" alt="image" src="https://user-images.githubusercontent.com/87339667/212123867-0908c285-cd6f-4c00-8404-56900382eab4.png">

About 60% feels less overwhelming:

<img width="1600" alt="image" src="https://user-images.githubusercontent.com/87339667/212124218-bd18e588-f844-4eee-b668-d0ce366e8da4.png">